core/src/main/java/com/linecorp/armeria/common/stream/DeferredStreamMessage.java

```diff
@@ -482,10 +482,8 @@ public CompletableFuture<List<T>> collect(EventExecutor executor, SubscriptionOp
     }
 
     private static SubscriptionImpl noopSubscription() {
-        final DefaultStreamMessage<?> streamMessage = new DefaultStreamMessage<>();
```

Comment:
We shouldn't use `DefaultStreamMessage` because `subscription.request` is called without subscribing.